sympy/concrete/summations.py

```diff
@@ -492,6 +492,16 @@ def is_convergent(self):
             except NotImplementedError:
                 pass
 
+        ### ----------- raabe test ---------------- ###
+        lim_val = Limit(sym * (sequence_term/sequence_term.subs(sym, sym + 1))
```

Comment:
for the case `log(n)/n**3` limit with abs returns unevaluated expression. So, did a work around. Calculated value of expression for lower and upper limit and check both are positive.